### PR TITLE
Fix table picker provider with parents

### PR DESCRIPTION
**Pull request by ausi**
Followup to #6504, #6510 and #6517

### CODE REVIEW

**Comment by ausi** [comment]
We should wait for @aschempp s approval

File: core-bundle/src/Picker/AbstractTablePickerProvider.php
Comment (aschempp) [inline core-bundle/src/Picker/AbstractTablePickerProvider.php:47]: this should early-return if `$config->getValue()` is empty in the first place. Maybe in the method or maybe here?

**Review by aschempp — CHANGES_REQUESTED** [review]

File: core-bundle/src/Picker/AbstractTablePickerProvider.php
Comment (ausi) [reply core-bundle/src/Picker/AbstractTablePickerProvider.php:47]: it does not execute a DB query if the value is empty.

**Review by aschempp — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**
